packages/provider/src/language-model/v1/language-model-v1-call-options.ts

```diff
@@ -21,11 +21,10 @@ The model can take this information and e.g. configure json mode, the correct
 low level grammar, etc. It can also be used to optimize the efficiency of the
 streaming, e.g. tool-delta stream parts are only needed in the
 object-tool mode.
-
-@deprecated mode will be removed in v2.
-All necessary settings will be directly supported through the call settings,
-in particular responseFormat, toolChoice, and tools.
    */
+  // deprecated: mode will be removed in v2.
+  // All necessary settings will be directly supported through the call settings,
+  // in particular responseFormat, toolChoice, and tools.
```

Comment:
removing this from the message that's displayed in the ide for now as it was confusing people and there is no v2 yet